game/src/Credits/Credits.gd

```diff
@@ -0,0 +1,98 @@
+extends Control
+
+signal backButtonPressed
+
+###############
+# Credits CSV format
+# The project title row is the only requirement within the csv file, however
+# it can be on any row, so long as it exists.
+# ----------------------
+# title,project-title
+# role-name,person-name
+# role-name,person-name
+# role-name,person-name
+# ...
+###############
+
+@export var coreCreditsPath : String = 'res://common/credits.csv'
+
+# REQUIREMENTS:
+# * 1.5 Credits Menu
+# * SS-17
+
+# REQUIREMENTS
+# * FS-4
+func loadCreditFile(path):
+	var roles = {}
+	var coreCredits = FileAccess.open(path, FileAccess.READ)
+	
+	while not coreCredits.eof_reached():
+		var line = coreCredits.get_csv_line()
+		
+		# If the line does not have an identifiable role or is empty then skip it
+		if line[0] == '':
+			continue
+		
+		var role: String = line[0].to_upper()
+		var person: String = line[1]
+		
+		if role not in roles.keys():
+			roles[role] = []
+			roles[role].push_back(person)
+		else:
+			roles[role].push_back(person)
+			
+	return roles
+
+# REQUIREMENTS:
+# * UI-34, UI-35
+func drawProjectCredits(project):
+	var projectParent = VBoxContainer.new()
+	var projectLabel = Label.new()
+	
+	projectParent.name = 'Credits-' + project['TITLE'][0]
+	projectLabel.name = 'Label-' + project['TITLE'][0]
+	projectLabel.text = project['TITLE'][0]
+	projectLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+	projectLabel.label_settings = load("res://src/Credits/ProjectLabelSettings.tres")
+	
+	for role in project.keys():
+		if role == 'TITLE':
+			continue
+		
+		var roleParent = VBoxContainer.new()
+		var roleLabel = Label.new()
+		
+		roleParent.name = 'Role-' + role
+		roleLabel.name = 'Label-' + role
+		roleLabel.text = role
+		roleLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+		roleLabel.label_settings = load("res://src/Credits/RoleLabelSettings.tres")
```

Comment:
Disregarding the next review comment, you should prefer preload over load as it takes no runtime cost, but also you shouldn't even be hardcoding the string if you can help it, you should always prefer export variables like so:

```gdscript
@export
var role_label_settings : LabelSettings
# ...
roleLabel.label_settings = role_label_settings
```

Reply:
Will take a look at it, thanks